Tests/SpeziViewsTests/SnapshotTests.swift

```diff
@@ -9,204 +9,389 @@
+            let lazyText = LazyText(verbatim: longString)
 
-#if canImport(WatchKit)
-        XCTAssertNotNil(eraser.wkImage)
-        XCTAssertNil(nonExistingImage.wkImage)
+#if os(iOS)
+            assertSnapshot(of: lazyText, as: .image(layout: .device(config: .iPhone13Pro)), named: "iphone-regular")
 #endif
+        }
 
-#if canImport(UIKit)
-        XCTAssertNotNil(eraser.uiImage)
-        XCTAssertNil(nonExistingImage.uiImage)
-#elseif canImport(AppKit)
-        XCTAssertNotNil(eraser.nsImage)
-        XCTAssertNil(nonExistingImage.nsImage)
+        @Test("Labeled Content")
+        func labeledContent() {
+            let labeledContent = LabeledContent("Clean") {
+                Text(verbatim: "Code")
+            }
+
+#if os(iOS)
+            assertSnapshot(of: labeledContent, as: .image(layout: .device(config: .iPhone13Pro)), named: "iphone-regular")
 #endif
+        }
+
+        @Test("Markdown View")
+        func markdownView() async throws {
+            let markdownView = MarkdownView(markdown: Data("*Clean* Coding".utf8))
+
+            try? await Task.sleep(nanoseconds: 1_000)
+
 
 #if os(iOS)
-        assertSnapshot(of: image, as: .image(layout: .device(config: .iPhone13Pro)), named: "iphone-regular")
+            assertSnapshot(of: markdownView, as: .image(layout: .device(config: .iPhone13Pro)), named: "iphone-regular")
 #endif
+        }
     }
 
-    @MainActor
-    func testTileHeaderLayout() {
+    // MARK: - Buttons
+    struct Buttons {
+        @MainActor
         struct TestView: View {
-            private let alignment: HorizontalAlignment
+            enum ButtonType: String {
+                case dismiss, info, async
+            }
+            let type: ButtonType
 
             var body: some View {
-                TileHeader(alignment: alignment) {
-                    Image(systemName: "book.pages.fill")
-                        .foregroundStyle(.teal)
-                        .font(.custom("Task Icon", size: 30, relativeTo: .headline))
-                        .dynamicTypeSize(...DynamicTypeSize.accessibility2)
-                        .accessibilityHidden(true)
-                } title: {
-                    Text("Clean Code")
-                } subheadline: {
-                    Text("by Robert C. Martin")
+                switch type {
+                case .dismiss:
+                    DismissButton()
+                case .info:
+                    InfoButton("Clean Code", action: {})
+                case .async:
+                    AsyncButton("Clean Code") {
+                        try? await Task.sleep(nanoseconds: 1_000)
+                    }
                 }
             }
 
-            init(alignment: HorizontalAlignment) {
-                self.alignment = alignment
+            init(type: ButtonType) {
+                self.type = type
             }
```

Comment:
This should allow you to get rid of all the `await`s  in the `@Test` arguments and allows to get rid of the suspension points there.

```suggestion
            nonsiolated init(type: ButtonType) {
                self.type = type
            }
```